src/main/java/game/racingcar/RacingCarController.java

```diff
@@ -1,30 +1,31 @@
 package game.racingcar;
 
 import game.racingcar.domain.Car;
-import game.racingcar.domain.Race;
+import game.racingcar.domain.GameCount;
+import game.racingcar.domain.RacingGame;
 import game.racingcar.view.InputView;
 import game.racingcar.view.OutputView;
 
 import java.util.List;
+import java.util.stream.Collectors;
 
 public class RacingCarController {
-    private static final String INIT_CAR_NAMES_MESSAGE = "경주할 자동차 이름을 입력하세요(이름은 쉼표(,)를 기준으로 구분)";
-    private static final String INIT_GAME_COUNT_MESSAGE = "시도할 회수는 몇회인가요?";
 
     public static void main(String[] args) {
-        String inputCarNames = InputView.getUserInput(INIT_CAR_NAMES_MESSAGE);
-        List<String> carNames = InputView.getCarNames(inputCarNames);
+        List<String> carNames = InputView.getCarNames();
+        List<Car> cars = carNames.stream().map(Car::new).collect(Collectors.toList());
```

Comment:
```suggestion
        List<Car> cars = carNames.stream()
                .map(Car::new)
                .collect(Collectors.toList());
```

stream 로직은 항상 가공 단위로 줄바꿈을 하는 것이 가독성이 좋습니다 :)
(다른 곳에서는 잘 해주셨네요 ㅎㅎ)